projects/addon-mobile/directives/dropdown-mobile/dropdown-mobile.directive.ts

```diff
@@ -19,15 +16,6 @@ import {TuiOptionMobile} from './option-mobile/option-mobile.component';
                     ? TuiDropdownMobileComponent
                     : inject(TUI_DROPDOWN_COMPONENT, {skipSelf: true}),
         },
-        {
-            provide: TUI_OPTION_CONTENT,
-            useFactory: () =>
-                inject(TUI_IS_MOBILE) &&
```

Comment:
Fix this example
https://taiga-previews-demo--pr11222-dropdown-mobile-demo-il2d3poo.web.app/components/select#dropdown-mobile

Different platforms - different appearance of checkbox
https://taiga-ui.dev/components/radio#platforms